### DISCUSSION TITLE
Go Pro 10/11 motion dection using mask

### DISCUSSION THREAD

**Discussion by suyunas** [gopro/labs#125] [Q&A]
I have GoPro 10/11 and using motion detection feature in which i am using a mask to capture movements. 
Starting its recording from the motion detection in masked area, GoPro would stop recording normally if there are no movements in the masked area. However, it wouldnot if there are movements in the non-masked area. 

Basically, I want the GoPro to stop recording when the subject leaves the masked area and enters the non-masked area. Kindly assist.

**Comment by dnewman-gpsw** [comment]
Yes, there was a bug discovered in the last release, which has no workaround. This should be addressed in a future Labs firmware update.
